.sync.yml

```diff
@@ -44,23 +28,21 @@ Gemfile:
           - mswin
           - mingw
           - x64_mingw
-      - gem: bolt
-        version: '~> 1.15'
-        condition: ENV['GEM_BOLT']
+      - gem: 'puppet_litmus'
+        platforms:
+          - ruby
+          - mswin
+          - mingw
+          - x64_mingw
+        condition: "ENV['PUPPET_GEM_VERSION'].nil? or ENV['PUPPET_GEM_VERSION'] !~ %r{ 5}"
+    ':development':
+      - gem: puppet-lint-i18n
```

Comment:
Is this needed @tphoney ?

Reply:
@eimlav Removed the system_tests group and executed testcase successfully